watcher.js

```diff
@@ -13,15 +13,18 @@ var conf = require('./configuration.js');
 var mail = require('./mail.js');
```

Comment:
Please revert the changes for now, I'll need to add some modifications for this as well tomorrow